aten/src/ATen/native/cuda/MultiTensorApply.cuh

```diff
@@ -95,15 +96,23 @@ struct FusedOptimizerTensorListMetadata {
   int start_tensor_this_launch;
 };
 
-template <typename T, typename U, typename... ArgTypes>
+template <typename T, typename U, typename index_t, typename... ArgTypes>
 C10_LAUNCH_BOUNDS_1(kBlockSize)
 __global__ void multi_tensor_apply_kernel(
     T tensorListMeta,
+    const index_t arg_of_index_t,
```

Comment:
The value of this argument doesn't mean a lot, the type does. This is why I defined two constexpr variables below.